data/json/items/containers.json

```diff
@@ -898,7 +900,8 @@
     "looks_like": "flask_glass",
     "description": "A 10ml laboratory cylindrical test tube, with a rubber stopper.",
     "weight": "36 g",
-    "volume": "11ml",
+    "volume": "92 ml",
```

Comment:
A 10ml flask is 92ml in volume? Surely the glass itself isnt 82ml? I think I am missing something.

Reply:
You're not missing anything. A 10 ml test tube is 100 mm x 13 mm. The volume of a cylinder of those dimensions is 13.2732 cm3 or aprox. 13 ml. God knows what numbers I entered in my calculator to get that result. 

Good catch, thanks.